Classes/Service/ImageResizer.php

```diff
@@ -310,7 +310,41 @@ public function processFile(
         if ((bool)$ruleset['auto_orient'] === true) {
             $orientation = ImageUtility::getOrientation($fileName);
             $isRotated = ImageUtility::isRotated($orientation);
-            $gifCreator->scalecmd = '-auto-orient ' . $gifCreator->scalecmd;
+
+            if ((bool)$ruleset['auto_orient_custom'] === true) {
+                switch ((int) $orientation) {
```

Comment:
no blank space before `$orientation` in CGL

Reply:
Yes, can be done, i leave that up to you :-)